Sources/Latte+UniversalLink.swift

```diff
@@ -0,0 +1,72 @@
+import Foundation
+import UIKit
+
+@available(iOS 13.0, *)
+public protocol LatteLinkHandler {
+    func handle(
+        context: UINavigationController,
+        authgear: Authgear,
+        handler: @escaping Latte.ResultHandler<Void>
+    )
+}
+
+@available(iOS 13.0, *)
+public extension Latte {
+    struct ResetLinkHandler: LatteLinkHandler {
+        let customUIEndpoint: String
```

Comment:
It seems we don't need store this; can replace `Authgear` parameter with `Latte` and get from it.

Reply:
@kiootic To support flows require an authenticated session, we will need an authgear instance. So I think we better keep `Authgear` and add `Latte`?

Reply:
`Latte` should contain a instance of `Authgear`?

Reply:
oh yes you are right.